pkg/cluster/context.go

```diff
@@ -42,16 +42,18 @@ import (
 
 // Context is used to create / manipulate kubernetes-in-docker clusters
 type Context struct {
-	name string
+	name             string
+	ControlPlaneMeta *ControlPlaneMeta
```

Comment:
in the long run, we might want to treat the Context object as an API and have public getters and setters instead of public fields.